rust-src/wallet_library/src/proofs.rs

```diff
@@ -0,0 +1,134 @@
+use anyhow::Result;
```

Comment:
Could you pleas not import this. It's ambiguous with Result in std.

Please use anyhow::Result